### PR TITLE
Add spider for seat_gb

### PR DESCRIPTION
**Pull request by abbensid2**

### CODE REVIEW

File: locations/spiders/seat_uk.py [hunk 1]
```diff
@@ -0,0 +1,25 @@
+import scrapy
+import xmltodict
+
+from locations.dict_parser import DictParser
+
+
+class SeatUkSpider(scrapy.Spider):
+    name = "seat_uk"
```
Comment (Cj-Malone) [inline locations/spiders/seat_uk.py:8]: Confusingly, the country code for the UK is actually GB.

**Comment by iandees** [comment]
The website property is missing the scheme ("https://") part of the URL. Can you add that in manually please?

**Comment by abbensid2** [comment]
> The website property is missing the scheme ("https://") part of the URL. Can you add that in manually please?

it's good

**Comment by abbensid2** [comment]
PR is closed, new PR: [4399](https://github.com/alltheplaces/alltheplaces/pull/4399)

**Pull request closed without merging**